cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -0,0 +1,124 @@
+from asgiref.sync import async_to_sync
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+from django.db import transaction
+
+from cl.lib.command_utils import logger
+from cl.lib.microservice_utils import microservice
+from cl.search.models import (
+    SOURCES,
+    Court,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    RECAPDocument,
+)
+
+
+def import_opinions_from_recap(court=None, total_count=0):
+    """Import recap documents into opinion db
+
+    :param court: Court ID if any
+    :param total_count: The number of new opinions to add
+    :return: None
+    """
+    count = 0
+    if not court:
+        courts = Court.objects.filter(jurisdiction=Court.FEDERAL_DISTRICT)
+    else:
+        courts = Court.objects.filter(id=court)
+    for court in courts:
+        logger.info(f"Importing RECAP documents for {court}")
+        cluster = (
+            OpinionCluster.objects.filter(docket__court=court)
+            .order_by("-date_filed")
+            .first()
+        )
+        if not cluster:
+            print("No clusters for this court")
+            logger.info(f"Court {court} has no opinion clusters")
+
+            continue
+        documents = RECAPDocument.objects.filter(
+            docket_entry__docket__court=court,
+            docket_entry__docket__date_filed__lte=cluster.date_filed,
+            is_available=True,
+            is_free_on_pacer=True,
+        ).order_by("docket_entry__docket__date_filed")
+
+        chunk_size = 100
+        paginator = Paginator(documents, chunk_size)
+        for page_number in paginator.page_range:
+            chunk = paginator.page(page_number).object_list
+            for recap_document in chunk:
+                docket = Docket.objects.get(
+                    docket_entries__recap_documents__id=recap_document.id
+                )
+                if "-cv-" not in docket.docket_number:
+                    logger.info(f"Skipping non civil opinion")
+                    continue
```

Comment:
You can't trust the dashes in docket numbers. They can be various types because people are nuts. I'd just check for cv, and I'd do case normalization to:

```suggestion
                if "cv" not in docket.docket_number.lower():
                    logger.info(f"Skipping non civil opinion") <---Also include an ID here, right?
                    continue
```